lib/square_mobile_payments_sdk_platform_interface.dart

```diff
@@ -30,12 +30,12 @@ abstract class SquareMobilePaymentsSdkPlatform extends PlatformInterface {
   
   }
 
-  Future<String> getSDKVersion() {
-    throw UnimplementedError('getSDKVersion() has not been implemented.');
+  Future<String> getSdkVersion() {
+    throw UnimplementedError('getSdkVersion() has not been implemented.');
```

Comment:
Should this file even be called?  Isn't the implementation in `square_mobile_payments_sdk_method_channel.dart`?